clayui.com/src/templates/docs.js

```diff
@@ -444,26 +437,15 @@ export const pageQuery = graphql`
 				}
 			}
 		}
-		allMarkdownRemark(filter: {fields: {slug: {nin: $blacklist}}}) {
+		allMarkdownRemark(filter: {fields: {slug: {in: $siblings}}}) {
 			edges {
 				node {
-					fields {
-						alwaysActive
-						layout
-						slug
-						title
-						order
-						indexVisible
-						draft
+					frontmatter {
+						tabName
 					}
+					html
 				}
 			}
 		}
```

Comment:
Consider keeping this query as is because we will use it for side navigation. You can check `<Sidebar />` in this file. Try to create a new query for the tabs.

```
tabs: allMarkdownRemark(filter: {fields: {slug: {in: $siblings}}}) {
	edges {
		node {
			fields {
				slug
				tabName
			}
			html
		}
	}
}
```